shell/platform/android/io/flutter/embedding/engine/FlutterShellArgs.java

```diff
@@ -133,4 +138,4 @@ public void remove(@NonNull String arg) {
     String[] argsArray = new String[args.size()];
     return args.toArray(argsArray);
   }
-}
\ No newline at end of file
+}
```

Comment:
Not sure why this line changed, but I'm guessing it's something to do with Vim. I tried to fix it but wasn't having much luck. If it's a problem, I'll go and fight with it some more.

Reply:
Your editor (or maybe git) is adding a newline at the end of the file.